stable/atlantis/templates/configmap.yaml

```
@@ -0,0 +1,14 @@
+{{- if .Values.serverSideConfig.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "atlantis.fullname" . }}
```

```suggestion
  name: {{ template "atlantis.fullname" . }}-repo-config
```

Probably best for this name to be suffixed with something in case we end up with more config maps